components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -1164,6 +1164,9 @@ def __init__(self, argument, index, args, argc, descriptorProvider,
 
         if not argument.variadic:
             if argument.optional:
+                # Treat undefined arguments as missing
+                undefined_condition = string.Template("!${args}.get(${index}).is_undefined()").substitute(replacer)
```

Comment:
Should I also change all other function calls to `.get()` to instead use `.index()`?